### PR TITLE
plan: add comments and tiny clean up

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: plan/physical_plan_builder.go [hunk 1]
```diff
@@ -172,9 +174,14 @@ func (p *DataSource) tryToGetDualTask() (task, error) {
 // convert2NewPhysicalPlan implements the PhysicalPlan interface.
 // It will enumerate all the available indices and choose a plan with least cost.
 func (p *DataSource) convert2NewPhysicalPlan(prop *requiredProp) (task, error) {
+	// If p is a inner plan in an IndexJoin, the IndexJoin will generate an inner plan by itself.
```
Comment (hanfei1991) [inline plan/physical_plan_builder.go]: an inner

File: plan/physical_plan_builder.go [hunk 2]
```diff
@@ -127,15 +128,17 @@ func (p *DataSource) tryToGetMemTask(prop *requiredProp) (task task, err error)
...
 		Table:       p.tableInfo,
 		Columns:     p.Columns,
 		TableAsName: p.TableAsName,
-	}.init(p.ctx)
+	}.init(p.ctx, p.stats)
 	memTable.SetSchema(p.schema)
 	memTable.Ranges = ranger.FullIntRange()
-	memTable.stats = p.stats
+
+	// Cancel push down these conditions.
```
Comment (hanfei1991) [inline plan/physical_plan_builder.go]: Stop to push down these ..

**Comment by hanfei1991** [comment]
@queenypingcap PTAL

**Comment by hanfei1991** [comment]
LGTM

**Comment by queenypingcap** [comment]
@lilin90 PTAL.

File: plan/stats.go [hunk 3]
```diff
@@ -35,7 +35,7 @@ func (s *statsInfo) Count() int64 {
 	return int64(s.count)
 }
 
-// collapse receives a selectivity and multiple it with count and cardinality.
+// scale receives a selectivity and multiple it with count and cardinality.
```
Comment (lilin90) [inline plan/stats.go]: The word "multiple" can be used as a noun or an adjective, but not a verb. Perhaps you mean "multiply" here?

Comment (zimulala) [reply plan/stats.go]: 1. Yes, Here needs a verb.
2. We need the `selectivity` here.

**Comment by zimulala** [comment]
PTAL @lilin90

**Comment by lilin90** [comment]
LGTM

**Pull request merged**